prefab_cloud_python/logger_client.py

```diff
@@ -39,6 +36,8 @@ def __init__(self, log_prefix=None, log_boundary=None, log_path_aggregator=None)
         self.log_boundary = log_boundary
         self.config_client = BootstrappingConfigClient()
         self.log_path_aggregator = log_path_aggregator
+        self.base_logger = structlog.get_logger()
+        self.base_logger._processors = PROCESSORS
```

Comment:
Have we confirmed that this only affects the processors of the logger created for this class?